code/game/objects/items.dm

```diff
@@ -158,7 +158,7 @@
 
 	forceMove(T)
 
-/obj/item/examine(mob/user)
+/obj/item/examine(mob/user, var/show_name = TRUE)
```

Comment:
Don't mix arguments. The second argument should correlate to the size argument of atom's examine.